net/src/in_memory/memory_server.rs

```diff
@@ -466,12 +466,12 @@ impl InMemoryServer {
         // Sender must be tracking
         let sender_info = Some((from_agent_id.clone(), Some(msg.request_id.clone())));
         let is_tracking =
-            self.priv_check_or_fail(&dna_address, &from_agent_id, sender_info.clone())?;
+            self.priv_check_or_fail(&dna_address.clone().into(), &from_agent_id, sender_info.clone())?;
```

Comment:
If this is never called with anything other than `dna_address.clone().into()` it might be worth changing the function so it doesn't need the `into` and/or `clone`

Reply:
I see the problem somewhere else. These newtype wrappers are causing use to call `.clone().into()` when we just want to pass references to data that's represented the same way. If we could convert references from one type to another, then we wouldn't need this boilerplate anyway. (Especially if we implemented `Deref`)